source/query-bar.txt

```diff
@@ -69,7 +69,7 @@ as in the following filter example:
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
 
 When you have entered a valid MongoDB query, the :guilabel:`Filter`
-label is no longer red and :guilabel:`Apply` button is enabled. 
+label is no longer red and :guilabel:`Find` button is enabled. 
```

Comment:
We usually don't specify the UI element unless it is unclear which element you mean (i.e. two elements named "Find": one link and one button).